frontend/packages/console-dynamic-plugin-sdk/src/app/components/utils/kebab.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { AccessReviewResourceAttributes } from '../../../extensions/console-types';
+
+export type KebabOption = {
```

Comment:
if this is just one type, then we don't need to have `kebab.tsx` here, we do have `console-types.ts` and `common-types.ts`